asistente_ladm_col/gui/qgis_model_baker/dlg_export_data.py

```diff
@@ -370,6 +380,12 @@ def update_configuration(self):
 
         configuration.xtffile = self.xtf_file_line_edit.text().strip()
 
+        # Set java path
+        settings = QSettings()
+        java_path = settings.value('QgisModelBaker/ili2db/JavaPath', '', str)
```

Comment:
Can we use `get_java_path_from_qgis_model_baker()`?

Reply:
Done. Thanks